tests/system/small/ml/test_llm.py

```diff
@@ -413,9 +413,17 @@ def test_llm_palm_score_params(llm_fine_tune_df_default_index):
     )
 
 
-@pytest.mark.flaky(retries=2)
-def test_llm_gemini_pro_score(llm_fine_tune_df_default_index):
-    model = llm.GeminiTextGenerator(model_name="gemini-pro")
+# test score() function for "gemini-pro" and "gemini-1.5" model
+@pytest.mark.parametrize(
+    "model_name",
+    (
+        "gemini-pro",
+        "gemini-1.5-pro-002",
+        "gemini-1.5-flash-002",
+    ),
+)
+def test_llm_gemini_score(llm_fine_tune_df_default_index, model_name):
```

Comment:
add for next test "test_llm_gemini_pro_score_params" as well

Reply:
done